config/config.go

```diff
@@ -19,12 +19,17 @@ type Config struct {
 	FilterAPIAuthToken       string        `envconfig:"FILTER_API_AUTH_TOKEN" json:"-"`
 	AWSRegion                string        `envconfig:"AWS_REGION"`
 	S3BucketName             string        `envconfig:"S3_BUCKET_NAME"`
+	S3PrivateBucketName      string        `envconfig:"S3_PRIVATE_BUCKET_NAME"`
 	CSVExportedProducerTopic string        `envconfig:"CSV_EXPORTED_PRODUCER_TOPIC"`
 	DatasetAPIURL            string        `envconfig:"DATASET_API_URL"`
 	DatasetAPIAuthToken      string        `envconfig:"DATASET_API_AUTH_TOKEN"`
 	ErrorProducerTopic       string        `envconfig:"ERROR_PRODUCER_TOPIC"`
 	GracefulShutdownTimeout  time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval      time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
+	VaultToken               string        `envconfig:"VAULT_TOKEN"                json:"-"`
```

Comment:
is the json tag to explicitly exclude this from the logging output?

Reply:
Yes, it will allow the log package to ignore the field when it is json marshalled here: https://github.com/ONSdigital/go-ns/blob/master/log/log.go#L112